src/main/java/frc/robot/subsystems/motorsubsystem/MotorSubsystem.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems.motorsubsystem;
+
+import frc.robot.hardware.interfaces.ControllableMotor;
+import frc.robot.subsystems.GBSubsystem;
+import frc.utils.calibration.sysid.SysIdCalibrator;
+
+public class MotorSubsystem extends GBSubsystem {
+
+	ControllableMotor motor;
+	GBSubsystem logPath;
+	MotorCommandsBuilder motorCommandsBuilder;
+	protected final SysIdCalibrator sysIdCalibrator;
+
+	public MotorSubsystem(GBSubsystem logPath, ControllableMotor motor) {
+		super(String.valueOf(logPath));
```

Comment:
```suggestion
		super(logPath);
```
You can just do this if it's a string, you don't need a GBSubsystem